admin-dev/themes/default/template/controllers/orders/form.tpl

```diff
@@ -494,14 +499,16 @@
         if (res.found) {
           var html = '';
           $.each(res.customers, function () {
+            var customer_view_url = customer_view_url_template.replace('/0/', '/' + this.id_customer + '/');
```

Comment:
Replace 0 by customer_id is a bad idea.
If you have a 0 elsewhere, it will be replaced.
Example: monsite.com/myshop0/

You must use something better for this kind of replace.
```js
customer_view_url_template.replace('/:customer_id:/', '/' + this.id_customer + '/');
```

Reply:
Well, I thought slashes (e.g. `/0/`) are good enough, don't you think?

What is more, I cannot generate url with anything else than `integer`, since customer id has route requirement for `\d+`.

Reply:
It's not enough, we already had a similar bug because of this :/
https://github.com/PrestaShop/PrestaShop/pull/10681/files#diff-3ea14175d71a90654ffa7f0bf0aea228R744

Reply:
It's not similar, as I understand `replace()` does not use regex, it replaces exact strings, so it tries to replace exactly `/0/` (with slashes). So it should work, unless you have something like `mysite.com/some/url/0/and/more/0/`, this way it would replace 2 parts.

Reply:
Ooooh, misreading because of slashes, thinking it was a regex ^^